app/scripts/modules/core/src/search/global/globalSearch.directive.js

```diff
@@ -2,9 +2,10 @@
 
 const angular = require('angular');
 const $ = require('jquery');
+const SPINNER = require('../../widgets/spinners/spinner.component').SPINNER;
 
 module.exports = angular.module('spinnaker.core.search.global.directive', [
-  require('./globalSearch.controller.js')
+  require('./globalSearch.controller.js'), SPINNER
```

Comment:
it is super weird (and not your fault) that this directive has both a link function and a controller in a separate file. 

It's more than three years old, so not that surprising, I guess.

(nothing to do, just looks odd now)